internal/common/compilation.bzl

```diff
@@ -17,6 +17,7 @@
 
 load(":common/module_mappings.bzl", "module_mappings_aspect")
 load(":common/json_marshal.bzl", "json_marshal")
+load("@io_bazel_rules_closure//closure/private:defs.bzl", "collect_js")
```

Comment:
does this require every user of rules_typescript to add rules_closure to their `WORKSPACE`? I don't think that's desirable, as many users will choose rollup/uglify instead. How can we avoid that?